modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -29,6 +34,60 @@ UnitTest.asynctest('browser.tinymce.core.EditorRemoveTest', (success, failure) =
       cRemoveEditor
     ])),
 
+    Logger.t('remove editor after stylesheets load', Chain.asStep({}, [
+      Chain.op(() => {
+        // setup monitoring for uncaught errors
+        uncaughtErrors = [];
+        window.addEventListener('unhandledrejection', recordError);
+      }),
+      Chain.control(
+        McEditor.cFromHtml('<textarea></textarea>', {
+          ...settings,
+          setup: (editor: Editor) => {
+            editor.on('LoadContent', () => {
+              // Hook the function called when stylesheets are loaded
+              // so we can destroy the editor right after starting to load them.
+              const realLoadAll = editor.ui.styleSheetLoader.loadAll;
+              editor.ui.styleSheetLoader.loadAll = (...args) => {
+                realLoadAll.apply(editor.ui.styleSheetLoader, args);
+                editor.destroy();
+              };
+            });
+          }
+        }), (f, value, next, die, logs) => {
+          f(value, (value2, logs2) => {
+            die('Expected editor would not load completely', logs2);
+          }, (err, logs2) => {
+            // As we have deliberately removed the editor during the loading process
+            // we have to intercept the error that is thrown by McEditor.cFromHtml.
+            if (err === McEditor.errorMessageEditorRemoved) {
+              next({}, logs2);
```

Comment:
```suggestion
              next(value, logs2);
```
I don't think we use it later, but would it be good practice to just pass through the original value in case someone else needs it?

Reply:
Done. Not that the value will be very useful since the editor is destroyed.